app/js/actions/jobs.js

```diff
@@ -55,14 +54,14 @@ export const linkInputArtifact = (input, artifacts) => ({
     artifacts
 });
 
-export const setJob = (action, actionType) => {
+export const setJob = action => {
```

Comment:
Do we want to make all single-arg actions parens-less like this, to keep it consistent? It does look a little nicer without the parens.